service/matching/config.go

```diff
@@ -22,6 +22,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:generate stringer -type loadCause -trimprefix loadCause -output loadcause_string_gen.go
```

Comment:
Nit: thinking if moving these to before the type definitions (lines 158, 159) will make it more likely to find by somebody new.

Reply:
`go:generate` has to be at the top of the file, before even `package` :(